haystack/nodes/file_converter/pdf.py

```diff
@@ -58,6 +58,7 @@ def __init__(
         :param multiprocessing: We use multiprocessing to speed up PyMuPDF conversion, you can disable it by setting it to False.
                                 If set to True (the default value), the total number of cores is used. To specify the number of cores to use, set it to an integer.
         """
+        # fitz_import.check()
```

Comment:
Why is this commented out?

Reply:
A leftover, thanks! As Massi pointed out in a previous PR, we don't need to check fitz here because the import is already checked in the `__init__.py` of the module.